core/src/main/resources/jenkins/telemetry/impl/Uptime/description.jelly

```diff
@@ -0,0 +1,11 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+    This trial collects two timestamps:
```

Comment:
Why collect both endpoints instead of just the difference? I guess this makes it easier to tell when a single correlator is somehow being used by multiple instances because of overlapping but non-equal endpoints. Are there any other reasons?

Reply:
> I guess this makes it easier to tell when a single correlator is somehow being used by multiple instances because of overlapping but non-equal endpoints.

That's the idea. Multiple submissions will have the same start time if and only if Jenkins wasn't restarted in between.